automation/test.sh

```diff
@@ -19,10 +19,30 @@
 
 set -ex
 
-export TARGET="windows2016"
+readonly TEMPLATES_SERVER="https://templates.ovirt.org/kubevirt/"
+
+export RHEL_NFS_DIR=${RHEL_NFS_DIR:-/var/lib/stdci/shared/kubevirt-images/rhel7}
+export RHEL_LOCK_PATH=${RHEL_LOCK_PATH:-/var/lib/stdci/shared/download_rhel_image.lock}
 export WINDOWS_NFS_DIR=${WINDOWS_NFS_DIR:-/var/lib/stdci/shared/kubevirt-images/windows2016}
 export WINDOWS_LOCK_PATH=${WINDOWS_LOCK_PATH:-/var/lib/stdci/shared/download_windows_image.lock}
-export KUBEVIRT_MEMORY_SIZE=12288M
+export KUBEVIRT_MEMORY_SIZE=16384M
+export KUBEVIRT_PROVIDER="os-3.11.0"
+export VERSION="v0.18.0"
```

Comment:
I wonder if we should extract this somewhere to make it easier to maintain and update. Could we autodetect the latest for example?

Reply:
Good idea, it should not be so hard to do it